### ISSUE TITLE
Error log emitted when client connected to relay proxy without events

### ISSUE THREAD

**Issue by chradcliffe** [launchdarkly/go-server-sdk#103]
**Is this a support request?** No

**Describe the bug**
When using the `ldcomponents.RelayProxyEndpointsWithoutEvents` to set the `Config.ServiceEndpoints` configuration parameter, the SDK emits an error log (from [here](https://github.com/launchdarkly/go-server-sdk/blob/v6/internal/endpoints/configure_endpoints.go#L86)):
```
ERROR: You have set custom ServiceEndpoints without specifying the Events base URI; connections may not work properly
```

**To reproduce**
Create a client that is connected to a relay proxy without events, e.g.:
```
	relayURI := "http://my-relay-hostname:8080"
	config := ld.Config{
	    ServiceEndpoints: ldcomponents.RelayProxyEndpointsWithoutEvents(relayURI),
	}
```

**Expected behavior**
An error log shouldn't be emitted for a regular configuration option.

**Logs**
N/A

**SDK version**
v6.1.0

**Language version, developer tools**
Go 1.22

**OS/platform**
Linux Alpine

**Additional context**
N/A

**Comment by cwaldren-ld** [comment]
Hi @chradcliffe , I agree this is a bug. Thanks for the report, I'll work on getting that fixed.

_Filed internally as 234826._

**Comment by cwaldren-ld** [comment]
Hi @chradcliffe , I'm targeting the fix at the `v7` major version of the SDK.

I wanted to check if you were planning on moving to v7 in the near term, knowing that v6 is unsupported in ~7 months? 

Otherwise, I will backport it to the v6 branch if you are finding this to be a critical issue.

**Comment by chradcliffe** [comment]
We can bump to v7 so I don't think we need a backport.

**Comment by cwaldren-ld** [comment]
Hi @chradcliffe , this has been fixed in [v7.1.1](https://github.com/launchdarkly/go-server-sdk/releases/tag/v7.1.1).
